recipes/OpenPatchMiner/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "OpenPatchMiner" %}
+{% set version = "0.1.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenPatchMiner-{{ version }}.tar.gz
+  sha256: 11b9089a0f55ec50b61fa002acf527299191af344aebb44053cbb021c8521355
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - keyring
+    - matplotlib-base
+    - numpy
+    - openslide-python
+    - pandas
+    - python >=3.6
+    - scikit-image
+    - tqdm
+    - twine
+    - wheel
```

Comment:
These seem like weird dependencies to have for the package...but they *are* what's listed upstream.

Reply:
Agreed - do you need me to change anything in this?

Reply:
If you do, it's going to fail `pip check`, so don't worry about it.

Reply:
Okay, so resolving.